src/features/filterTodo.ts

```diff
@@ -0,0 +1,33 @@
+import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
+
+import { FilterByParameters } from '../utils/filterByParameters';
+
+export const filterTodo = {
+  getTodos: (state: RootState) => state.todos,
+  getFilteredTodos: (state: RootState) => {
+    const { todos, filter } = state;
+    const { query, status } = filter;
+
+    let filteredTodos: Todo[];
+
+    switch (status) {
+      case FilterByParameters.Copleted:
```

Comment:
There is a typo in 'FilterByParameters.Copleted', it should be 'FilterByParameters.Completed'.